doc/nrf/release_notes.rst

```diff
@@ -5,19 +5,20 @@ Release Notes
 
 See the release notes for information about specific |NCS| releases.
 
-Important issues that are discovered after a release is tagged are listed in the `Known issues`_ wiki page.
+Important issues that are discovered after a release is tagged are listed on the :ref:`known_issues` page.
 
 .. important::
    A "99" at the end of the version number of this documentation indicates continuous updates on the master branch since the previous major.minor release.
-   Changes between releases that are merged into the master branch are tracked in the `nRF Connect SDK changelog`_ in the sdk-nrf repository wiki.
+   Changes between releases that are merged into the master branch are tracked on the :ref:`ncs_release_notes_latest` page.
```

Comment:
This depends on the decision where we want to keep the changes, same as the changes to the release-notes-latest file.

Reply:
Decision will be taken after release.